ibm_watson.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "concurrent-ruby", "~> 1.0"
   spec.add_runtime_dependency "eventmachine", "~> 1.2"
   spec.add_runtime_dependency "faye-websocket", "~> 0.11"
-  spec.add_runtime_dependency "http", "~> 4.1.0"
+  spec.add_runtime_dependency "http", "~> 4.4.0"
   spec.add_runtime_dependency "ibm_cloud_sdk_core", "~> 1.1.1"
```

Comment:
do we need to update the version of the core to match the latest release?  https://github.com/IBM/ruby-sdk-core/releases/tag/v1.1.3

Reply:
so thats a good question.  It does bring in 1.1.3 now by leaving it at 1.1.1.  I guess the gems work by getting the latest fix release.  But I think thats a good idea to move it to 1.1.3...Let me make the change and run some tests to make sure it holds..

Reply:
I think that is a good change.  All the tests passed.  Thank you!!